eq-author/src/components/Toasts/ToastContainer/index.js

```diff
@@ -38,15 +30,15 @@ const ToastWrapper = styled.div`
   justify-content: center;
 `;
 
-export const ToastArea = ({ toasts, dismissToast }) => {
+export const ToastArea = ({ toasts, onDismissToast }) => {
   return (
     <ToastWrapper>
       <ToastOuterContainer>
         <ToastInnerContainer>
```

Comment:
Does `ToastWrapper`, `ToastOuterContainer` and `ToastInnerContainer` only wrap the list of toasts? If so, maybe call them `ToastsWrapper` or `ToastListWrapper` so it is more clear that they are not used for the toast itself?

Reply:
Yeah, I'll update the names it is a bit confusing. I might drop the toast prefix if it still makes sense after.